### PR TITLE
throttling twitter file import

### PR DESCRIPTION
**Pull request by ymatsuda**
@eishay @andrewconner

### CODE REVIEW

**Comment by eishay** [comment]
:+1:

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepInterner.scala [hunk 1]
```diff
@@ -173,7 +173,7 @@ class KeepInterner @Inject() (
       }
       if (uri.state == ACTIVE || uri.state == INACTIVE) {
         val date = source match {
-          case KeepSource.bookmarkImport => currentDateTime.plus(Random.nextInt(MAX_RANDOM_SCHEDULE_DELAY))
+          case KeepSource.bookmarkImport | KeepSource.twitterFileImport => currentDateTime.plus(Random.nextInt(MAX_RANDOM_SCHEDULE_DELAY))
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepInterner.scala:176]: How about `case s if KeepSource.imports.contains(s)`? Covers more cases.

**Comment by andrewconner** [comment]
Thanks!

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepInterner.scala (hunk 1 above)
Comment (ymatsuda) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepInterner.scala:176]: Good idea.

**Pull request merged**